port/common/omrsysinfo.c

```diff
@@ -172,7 +172,7 @@ omrsysinfo_get_processor_feature_name(struct OMRPortLibrary *portLibrary, uint32
  * @param[out] buffer The processor feature output string.
  * @param[in] length The size of the buffer in number of bytes.
  */
-void
+intptr_t
```

Comment:
I don't know what the convention here is but since we duplicated the documentation here, we should also add the `@return` here as well.